frontend/src/components/SystemSubmitDrawer/index.tsx

```diff
@@ -626,17 +628,35 @@ export function SystemSubmitDrawer(props: Props) {
           </Form.Item>
 
           <Form.Item
-            name="metric_names"
             label="Metrics"
+            name="metric_names"
```

Comment:
The `name` property only works if it is specified on the `<Form.Item />` wrapping the input component (in this case `<Select />`). I think moving it to the `<Form.Item />` on line 638 would fix this issue. Could you please double-check and make sure system submission still works after the change?